minifi/minifi-c2/minifi-c2-assembly/pom.xml

```diff
@@ -26,7 +26,7 @@ limitations under the License.
     <packaging>pom</packaging>
     <description>This is the assembly of Apache MiNiFi's - Command And Control Server</description>
     <properties>
-        <minifi.c2.server.port>10080</minifi.c2.server.port>
+        <minifi.c2.server.port>10090</minifi.c2.server.port>
```

Comment:
The corresponding [Dockerhub](https://github.com/apache/nifi/blob/7a16619969a90655b5aed6bf938fa0a3019ab673/minifi/minifi-c2/minifi-c2-docker/dockerhub/Dockerfile#L50) and [Dockermaven](https://github.com/apache/nifi/blob/7a16619969a90655b5aed6bf938fa0a3019ab673/minifi/minifi-c2/minifi-c2-docker/dockermaven/Dockerfile#L41) Dockerfiles also need to be updated.